### PR TITLE
Added LearnAwesome under open-source projects

### PR DESCRIPTION
**Pull request by eshnil**
Wanted to add a direct link to the site as well: https://learnawesome.org
But kept only the link to code repo in line with other existing listings.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -210,6 +210,7 @@
 - [Statusfy](https://github.com/bazzite/statusfy) - Status page system using Tailwind CSS.
 - [Todolist](https://github.com/guillaumebriday/todolist-frontend-vuejs) - To-do list application using Tailwind CSS.
 - [LeagueStats](https://github.com/vkaelin/LeagueStats) - Statistics website for League of Legends players.
+- [LearnAwesome](https://github.com/learn-awesome) - Humanity's universal learning map (built with Tailwind CSS)
```
Comment (joshmanders) [inline readme.md:213]: This needs to be at the bottom of the list. Please read [item 3 on the guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/contributing.md).

**Comment by joshmanders** [comment]
ping @eshnil

**Comment by joshmanders** [comment]
Closing as there was no response.

**Comment by frederikhors** [comment]
Please re-open, @joshmanders.

**Comment by joshmanders** [comment]
@frederikhors for what reason? The author is MIA.

**Pull request closed without merging**